lib/Public/ContentManager.php

```diff
@@ -33,6 +36,16 @@ public function __construct(
 	) {
 	}
 
+	/**
+	 * Checks if the context chat app is enabled or not
+	 *
+	 * @return bool
+	 * @since 4.4.0
+	 */
+	public function isContextChatAvailable(): bool {
+		return true;
```

Comment:
```suggestion
		return $this->config->getAppValue(Application::APP_ID, 'backend_init', 'false') === 'true';
```
maybe it makes sense to do this as well? It is set when at least one request to the backend is done from the context chat php to the backend app.
from: https://github.com/nextcloud/context_chat/blob/main/lib/Service/LangRopeService.php

Reply:
Makes sense in the off chance that the backend isn't installed yet.